locales/en.yml

```diff
@@ -19,6 +20,9 @@ en:
           comment:
             syntax: jira comment on <issue> <comment text>
             desc: Adds <comment text> to <issue>
+          point:
+            syntax: jira point <issue> as <points>
+            desc: Adds <points> Story Points to <issue>
```

Comment:
Is the Story Points field specific to our JIRA organization? If so, you may want to change this template to something more generic like "points"